src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -71,19 +72,31 @@ const SubsidiesSummary = ({
     return null;
   }
 
+  const isOneAcademy = enterpriseCustomer?.enableOneAcademy;
   const searchCoursesCta = (
     !programProgressPage && !enterpriseCustomer.disableSearch && showSearchCoursesCta && (
       <Button
         as={Link}
-        to={`/${enterpriseCustomer.slug}/search`}
+        to={isOneAcademy ? `/${enterpriseCustomer.slug}/academies/${academies[0].uuid}` : `/${enterpriseCustomer.slug}/search`}
```

Comment:
Can we please add optional chaining here to prevent any errors related to academies or academies[0] being undefined?

```
to={isOneAcademy && academies?.[0]?.uuid 
    ? `/${enterpriseCustomer.slug}/academies/${academies[0].uuid}` 
    : `/${enterpriseCustomer.slug}/search`}
```